api/src/main/java/com/aldoborrero/tinder/api/model/TinderEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package com.aldoborrero.tinder.api.model;
+
+import retrofit.Endpoint;
+
+public class TinderEndpoint implements Endpoint {
```

Comment:
@aldoborrero I think that we cannot obligate to the user with the production endpoint because what it happens if the endpoint was changed it by Tinder. I think that we can provide of this object TinderEndpoint and inside we can write the constants of the production endpoint by if the user want to change it. He can do it. 

What do you think?